index.html

```diff
@@ -51,7 +51,70 @@ <h2 class="describe">I’am a Software Developer</h2>
         </ul>
         
     </section>
-    
+    <section class="works">
+        <div class="title">
```

Comment:
- [ ] Kindly consider using the same spacing(margin, padding) as the design in Figma.
- [ ] Also consider using the correct path for your social media icons. as you can how yours are broken
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/198566827-994bedd1-8bde-4902-bb5b-014859e35191.png)


Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/198566940-6a03d8cf-731b-4f32-bab6-3e9fa80e4c55.png)